fbb/header.go

```diff
@@ -137,7 +137,7 @@ func (h Header) Write(w io.Writer) error {
 // (See DecodeHeader for mime.WordDecoder differences).
 type WordDecoder struct{ mime.WordDecoder }
 
-// Decode decodes an encoded-word.
+// DecodeHeader decodes decodes an encoded-word.
```

Comment:
"... decodes decodes ..."

Heh, I can understand why your bot was confused by this.. but I won't merge it.